# The list of Yolo classified objects
perception/YoloObject[] objects
